resource-definitions/template-driver/horizontalpodautoscaler/hpa-template.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: hpa-template
+entity:
+  driver_type: humanitec/template
+  name: hpa-template
+  type: horizontal-pod-autoscaler
+  driver_inputs:
+    values:
+      templates:
+        init: |
+          workload: {{ index (splitList "." "${context.res.id}") 1 }}
+          maxReplicas: {{ .resource.maxReplicas | default 3 }}

```

As an example, you could add the min() clamp here as well?

```suggestion
          maxReplicas: {{ .resource.maxReplicas | default 3 | min 10 }}

```